src/Microsoft.ML.Data/Transforms/CategoricalsCatalog.cs

```diff
@@ -28,4 +28,30 @@ public static KeyToValueEstimator ToValue(this TransformsCatalog.CategoricalTran
         public static KeyToValueEstimator ToValue(this TransformsCatalog.CategoricalTransforms catalog, params (string input, string output)[] columns)
              => new KeyToValueEstimator(CatalogUtils.GetEnvironment(catalog), columns);
     }
+
+    /// <summary>
+    /// Extensions for KeyToVectorMappingEstimator.
+    /// </summary>
+    public static class ToVectorCatalog
+    {
+        /// <summary>
+        /// Convert the key types back to their original vectors.
+        /// </summary>
+        /// <param name="catalog">The categorical transform's catalog.</param>
+        /// <param name="columns">The input column to map back to vectors.</param>
+        public static KeyToVectorMappingEstimator ToVector(this TransformsCatalog.CategoricalTransforms catalog,
```

Comment:
>CategoricalTransforms [](start = 82, length = 21)

Actually I was expecting `ToKey`, `ToVector`, `ToBinaryVector` and `ToValue` to be in `Conversions`, not `Categorical`. In any case, let's not shorten it too much: `MapKeyToVector`, `MapValueToKey` etc. would be better I think

Reply:
would the respective Estimator live in the Conversions namespace?


---
In reply to: [229773742](https://github.com/dotnet/machinelearning/pull/1460#discussion_r229773742) [](ancestors = 229773742)

Reply:
Personally I would vote to keep them in the Categorical, if the opposite operation  (Term) is there. 

---
In reply to: [229779270](https://github.com/dotnet/machinelearning/pull/1460#discussion_r229779270) [](ancestors = 229779270,229773742)